# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/palemoon/hotfix-xpcstubs.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- Pale-Moon/platform/xpcom/reflect/xptcall/md/unix/moz.build.vanilla	2025-08-20 17:15:48.993781959 +0200
+++ Pale-Moon/platform/xpcom/reflect/xptcall/md/unix/moz.build	2025-08-20 17:16:12.748704827 +0200
@@ -170,7 +170,7 @@
         'xptcstubs_ppc_openbsd.cpp',
     ]
 
-if CONFIG['OS_ARCH'] == 'Linux' and CONFIG['OS_TEST'] == 'sparc':
+if CONFIG['OS_ARCH'] == 'Linux' and CONFIG['OS_TEST'] in ('sparc','sparcv9'):
     SOURCES += [
         'xptcinvoke_asm_sparc_linux_GCC3.s',
         'xptcinvoke_sparc_solaris.cpp',
